README.md

```diff
@@ -181,8 +183,28 @@ To delete and rebuild TAPscan from the data in the `_data` folder, you can use `
 
 Steps to follow to make data ready for inclusion in TAPscan:
 
+### Species Lettercode
+
+Make sure that each of your species has a unique (uppercase) lettercode. This lettercode must be consistent in the species information file, and the FASTA files.
+
 ### Sequence Files
-Create a fasta file per species, place it in the `_data/fasta` folder, named by their lettercode, e.g. `ACTCH.fa, ZYGCI.fa` etc.
+Create a fasta file per species, place it in the `_data/fasta` folder, named by their lettercode, e.g. `ACTCH.fa, ZYGCI.fa` etc. Every header inside the fasta file must also begin with the lettercode followed by an underscore, e.g. `>ACTCH_AA512007.0`. Everything after the underscore is assumed to be protein identifier, and will be used to search on PLAZA.
+
+In order to retain information about the organel/source of origin, a suffix (e.g. `_mt` for mitochondria) may be added to the FASTA headers, for example `>ACTCH_mt_AA512007.0`
+
+Valid suffixes are:
+
+```
+_mt : mitochondria
+_pl : plasmids
+_pt : plastids
+_hc :
+_hl :
+_chl : chloroplast
+_tr : transcriptome
+_lc :
+_org :
```

Comment:
did I miss any suffixes, and can you fix/add what each of them stands for?

Reply:
@shiltemann,

I added the information. but what is this `_hl :`. I removed it for now. For which species, you have this kind of header prefix?

Reply:
sorry, think thats a typo from chl, please ignore :)